### DISCUSSION TITLE
running behind digital ocean network loadbalancer

### DISCUSSION THREAD

**Discussion by oscargoldman** [pocketbase/pocketbase#5241] [Q&A]
I have pocketbase running on a linux droplet on digital ocean, and using the public IP in the serve --http flag, I can connect without issues. I'd like to put this droplet behind a load balancer, mainly because digital ocean load balancers terminate with SSL using lets encrypt and manage that automatically. I have tested the load balancer and it's connecting to the droplet, but can't seem to tunnel through to pocketbase. What is the correct http or https value for serve in this case?

**Comment by oscargoldman** [comment]
reading a little more about this, I found that using serve with the domain name pointing to the droplet automagically generates a lets encrypt cert and serves pocketbase with ssl, so I've removed the load balancer and deleted the certificate there. However, I can't reach pocket base at mydomain.com/_/ or /api/ . The server logs this error: 2024/07/21 19:06:47 http: TLS handshake error from 205.210.31.88:56450: tls: client offered only unsupported versions: [302 301]

**Reply by ganigeorgiev** [reply]
Either you are using some very old client/browser (_302/0x302 and 301/0x301 codes usually refers to TLS 1.1 and TLS 1.0 which are considered insecure and deprecated and we require min TLS 1.2_) or there is something else wrong with the droplet setup.

It could be also a client/browser cache of previous failed access attempts I'm not sure.

If you are still not able to resolve it, I recommend to start with a clean droplet and follow the [Minimal deployment setup](https://pocketbase.io/docs/going-to-production/#minimal-setup) instructions.

**Comment by oscargoldman** [comment]
Thanks for your response. I think it was actually a DNS delay. Since I'd issued a cert for the load balancer, once I deleted them both, and issued again directly for the droplet, after about an hour things just started to work. Thanks again - really liking the platform so far.

**Reply by bharathNewstreettech** [reply]
Hi oscargoldman your domain was hosted on any dns provider? or digital ocean?
